server/controllers/request.js

```diff
@@ -0,0 +1,75 @@
+require('dotenv').config();
+const Request = require('../models/Request');
+const asyncHandler = require('express-async-handler');
+
+// @route GET /request
+// @desc get requests related to a logged in user or sitter
+// @access Private
+exports.requestGet = asyncHandler(async (req, res) => {
+  const userId = req.user.id;
+
+  const getUser = await Request.find({userId: userId})
+  //
+  if(getUser != null){
```

Comment:
Couldn't this just be `if (getUser) { . . . }`?

Also, please clean up the code style.

Reply:
Change throughout the code